lib/toc_languages.py

```diff
@@ -179,10 +179,8 @@ def join_exercises(self, key, lang1, c1, lang2, c2):
                 override = self.app.env.config.override.get(c1.get('category'), {})
                 if k in override:
                     c[k] = override[k].format(key=key)
-                elif v != c2.get(k, v):
```

Comment:
Any reason why this `elif` was removed? It's not obvious to me. Is this related to the new gitmanager options?

Reply:
I believe this is because the url may contain the exercise key with the language, so they need to be joined. The join_keys will still raise an exception if the URLs are still different after removing the language tag.